plugins/godot-gradle-plugin/src/main/kotlin/godot/gradle/Platform.kt

```diff
@@ -0,0 +1,10 @@
+package godot.gradle
+
+enum class Platform {
```

Comment:
As said in `GodotPlugin.kt` some targets seem to be missing

Reply:
Yeah, nice catch. I forgot about arm and non-arm variants for mobile.